pcsx2-qt/MainWindow.cpp

```diff
@@ -1454,6 +1456,9 @@ void MainWindow::onGameListEntryContextMenuRequested(const QPoint& point)
 			connect(menu.addAction(tr("Check Wiki Page")), &QAction::triggered, [this, entry]() { goToWikiPage(entry); });
 		}
 
+		action = menu.addAction(tr("Open Screenshots Folder"));
+		action->setEnabled(EmuConfig.GS.OrganizeScreenshotsByGame);
```

Comment:
what's the point of adding a disabled entry in a context menu based on a config option? it's the user's decision to enable/disable the feature, so if I disable it I don't want the menu to be cluttered with irrelevant actions.

IMHO either don't add the action when `OrganizeScreenshotsByGame` is disabled or make the action work regardless of `OrganizeScreenshotsByGame`, e.g. by opening the generic snaps folder.